rust-clippy.md

```diff
@@ -0,0 +1,7 @@
+title: clippy
+repo: rust-lang/rust-clippy
+graphs:
+  pr-status: Pull requests status
+  pr-activity: Last activity on pull requests
+  pr-age: Pull requests creation dates
+layout: graphs
```

Comment:
This needs `---` at the start and the end of the file to mark it as the frontmatter.

Reply:
Oops, updated. :sweat_smile: